docs/examples/faq.ipynb

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Do you plan to make an FAQ page specific for the example (and thus other sections)? Or should it be in a more generic location?

Reply:
It's supposed to be generic, not sure where to put it.
A section of its own? Change the `Reference` section to something else and put it next to the API reference?

Reply:
Scipp has it in the `Getting Started` section: https://scipp.github.io/getting-started/faq.html